src/sentry/static/sentry/app/components/stream/group.tsx

```diff
@@ -424,7 +424,7 @@ class StreamGroup extends React.Component<Props, State> {
           <AssigneeSelector id={data.id} memberList={memberList} />
         </AssigneeWrapper>
         {canSelect && hasInbox && (
-          <ActionsWrapper className="visible-lg">
```

Comment:
Is this class name in use elsewhere? can we remove the css def too?

Reply:
yeah we can, will follow up